src/imap.h

```diff
@@ -27,6 +27,11 @@ class IMap
     return map_type_ == BPF_MAP_TYPE_PERCPU_HASH ||
            map_type_ == BPF_MAP_TYPE_PERCPU_ARRAY;
   }
+  bool is_not_clearable() const
```

Comment:
Can we name this `is_clearable()`? Would be confusing to read `!is_not_clearable()` later on (the double negation).

Reply:
That makes sense. will fix.

Reply:
done.